pkg/storage/bloom/v1/fuse.go

```diff
@@ -331,23 +331,16 @@ func (fq *FusedQuerier) runSeries(schema Schema, series *SeriesWithMeta, reqs []
 				continue
 			}
 
-			// TODO(owen-d): copying this over, but they're going to be the same
```

Comment:
First, we need to check if the field name is part of the list of indexed fields, which is exposed in the `series *SeriesWithMeta`